backend/datasources/dummy.py

```diff
@@ -362,6 +414,9 @@ def invoice_list(self, username: str) -> list[Invoice]:
     def invoice_pdf(self, username: str, invoice_number: str) -> InvoicePdf:
         return dummy_invoice_pdf(username, invoice_number)
 
+    def invoices_zip(self, username: str, invoice_numbers: []) -> InvoicesZip:
```

Comment:
and in general in all the stack `invoice_numbers` as `list[str]`

Reply:
done